### PR TITLE
Adds Topic() calls to get and adjust metacoins

### PR DESCRIPTION
**Pull request by ike709**
Crossed asked for it.

## Changelog
:cl:
server: Added Topic() calls for getting/adjusting metacoins.
/:cl:

### CODE REVIEW

File: code/datums/world_topic.dm [hunk 1]
```diff
@@ -232,3 +232,49 @@
...
+	return mc_count ? text2num(mc_count) : null
+
+/datum/world_topic/adjust_metacoins
+	keyword = "adjust_metacoins"
+	require_comms_key = TRUE
+
+/datum/world_topic/adjust_metacoins/Run(list/input, addr)
+	var/ckey = input["ckey"]
+	var/amount = input["amount"]
+	var/log_id = input["id"]
```
Comment (Crossedfall) [inline code/datums/world_topic.dm]:
```suggestion
	var/adjuster_ckey = input["id"]
```

File: code/datums/world_topic.dm [hunk 2]
```diff
@@ -232,3 +232,49 @@
...
+	. = TRUE
+
+	var/datum/DBQuery/query_metacoins = SSdbcore.NewQuery(
+		"UPDATE [format_table_name("player")] SET metacoins = metacoins + :amount WHERE ckey = :ckey",
+		list("amount" = amount, "ckey" = ckey)
+	)
+	if(!query_metacoins.warn_execute())
+		. = FALSE
+
+	log_game("[ckey]'s metacoins were adjusted ([amount > 0 ? "+[amount]" : "[amount]"]) via Topic() call by [log_id ? "[log_id]" : "Unknown"]")
```
Comment (Crossedfall) [inline code/datums/world_topic.dm]:
```suggestion
	log_game("[ckey]'s metacoins were adjusted ([amount > 0 ? "+[amount]" : "[amount]"]) via Topic() call by [adjuster_ckey ? "[adjuster_ckey]" : "Unknown"]")
```

**Review by qwertyquerty — DISMISSED** [review]

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Overall code looks good. Only change I'd make is to update the `log_id` variable to be something more descriptive.

**Review by qwertyquerty — APPROVED** [review]

**Review by Crossedfall — APPROVED** [review]

**Comment by MCterra10** [comment]
Metacoins crosscomms 😳

**Pull request merged**